docs/Developers/Design/CRD_design.md

```diff
@@ -0,0 +1,69 @@
+This document describes the set of [Custom Resource Definitions
+(CRDs)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+that will be used to deploy mixins for a storage system. The actual implementation
+of the resources described here will be phased in during development. The
+purpose of this document is to provide the overall structure, ensuring the end
+result provides necessary configurability in a user-friendly manner.
+
+# Overview
+
+A single Monstorak operator can take care of deployment of mixins for one or
```

Comment:
~A single Monstorak operator can take care~ -> `An instance of Monstorak operator takes care`

Reply:
Ack